tests/test_data_importer.py

```diff
@@ -47,36 +50,44 @@ def data_dir():
 
 
 @pytest.mark.parametrize(
-    "dataset",
+    "importer,dataset",
     [
-        "mtdata_Neulab-tedtalks_test-1-eng-rus",
-        "opus_ELRC-3075-wikipedia_health/v1",
-        "flores_dev",
-        "sacrebleu_wmt19",
+        ("mtdata", "Neulab-tedtalks_test-1-eng-rus"),
+        ("opus", "ELRC-3075-wikipedia_health_v1"),
+        ("flores", "dev"),
+        ("sacrebleu", "wmt19"),
     ],
 )
-def test_basic_corpus_import(dataset, data_dir):
-    prefix = data_dir.join(dataset)
-    output_src = f"{prefix}.{SRC}.{ARTIFACT_EXT}"
-    output_trg = f"{prefix}.{TRG}.{ARTIFACT_EXT}"
-
-    run_import("corpus", dataset, prefix)
+def test_basic_corpus_import(importer, dataset, data_dir):
+    data_dir.run_task(
+        f"dataset-{importer}-{dataset}-en-ru",
+        env={
+            "COMPRESSION_CMD": COMPRESSION_CMD,
+            "ARTIFACT_EXT": ARTIFACT_EXT,
+            "WGET": os.path.join(CURRENT_FOLDER, "fixtures/wget"),
+            "MOCKED_DOWNLOADS": get_mocked_downloads(),
+        },
+    )
+
+    prefix = data_dir.join(f"artifacts/{dataset}")
+    output_src = f"{prefix}.ru.gz"
+    output_trg = f"{prefix}.en.gz"
 
     assert os.path.exists(output_src)
     assert os.path.exists(output_trg)
     assert len(read_lines(output_src)) > 0
     assert len(read_lines(output_trg)) > 0
 
 
-@pytest.mark.parametrize(
-    "params",
-    [
-        ("sacrebleu_aug-upper_wmt19", is_upper_case, AUG_MIN_RATE, AUG_MAX_RATE),
-        ("sacrebleu_aug-upper-strict_wmt19", is_upper_case, 1.0, 1.0),
-        ("sacrebleu_aug-title_wmt19", is_title_case, AUG_MIN_RATE, AUG_MAX_RATE),
-        ("sacrebleu_aug-title-strict_wmt19", is_title_case, 1.0, 1.0),
-    ],
-)
+augmentation_params = [
+    ("sacrebleu_aug-upper_wmt19", is_upper_case, AUG_MIN_RATE, AUG_MAX_RATE),
+    ("sacrebleu_aug-upper-strict_wmt19", is_upper_case, 1.0, 1.0),
+    ("sacrebleu_aug-title_wmt19", is_title_case, AUG_MIN_RATE, AUG_MAX_RATE),
+    ("sacrebleu_aug-title-strict_wmt19", is_title_case, 1.0, 1.0),
+]
+
+
+@pytest.mark.parametrize("params", augmentation_params, ids=[d[0] for d in augmentation_params])
```

Comment:
This is a drive-by, but I'm trying to make the keys for tests readable. Previously they were all just `params0` and `params1`, etc.

Now they are:

```
tests/test_data_importer.py::test_basic_corpus_import[mtdata-Neulab-tedtalks_test-1-eng-rus] PASSED                                                                                 [ 11%]
tests/test_data_importer.py::test_basic_corpus_import[opus-ELRC-3075-wikipedia_health_v1] PASSED                                                                                    [ 22%]
tests/test_data_importer.py::test_basic_corpus_import[flores-dev] PASSED                                                                                                            [ 33%]
tests/test_data_importer.py::test_basic_corpus_import[sacrebleu-wmt19] PASSED                                                                                                       [ 44%]
tests/test_data_importer.py::test_specific_augmentation[sacrebleu_aug-upper_wmt19] PASSED                                                                                           [ 55%]
tests/test_data_importer.py::test_specific_augmentation[sacrebleu_aug-upper-strict_wmt19] PASSED                                                                                    [ 66%]
tests/test_data_importer.py::test_specific_augmentation[sacrebleu_aug-title_wmt19] PASSED                                                                                           [ 77%]
tests/test_data_importer.py::test_specific_augmentation[sacrebleu_aug-title-strict_wmt19] PASSED                                                                                    [ 88%]
tests/test_data_importer.py::test_augmentation_mix PASSED                                                                                                                           [100%]
```